pkg/metrics/metrics.go

```diff
@@ -372,6 +388,13 @@ func (m *ServerMetrics) InitSchedule(scheduleName string) {
 	// -------------------------------------------------------------------
 }
 
+// SetBackupCountGauge records the existing number of backups
+func (m *ServerMetrics) SetBackupCountGauge(size int) {
```

Comment:
I think this function could just increment a counter, rather than take a size. That would be similar to `RegisterBackupAttempt`.